litentry/litentry-parachain
# store enclave public info in teerex

relevant issue:
https://github.com/litentry/litentry-parachain/issues/1019

---

- [x] update `SgxStatus`
- [x] add ias-verify test
- [x] store enclave-related info in teerex

---

test teerex with real enclave info :
1. disable `tee-dev` feature in `node/Cargo.toml`
2. build `parachain` without feature `skip-ias-check`
3. run tee-worker without flag `skip-ra`
